src/libDirectoryService/DSBlockPostProcessing.cpp

```diff
@@ -231,8 +230,7 @@ void DirectoryService::UpdateMyDSModeAndConsensusId()
                              << "] DSBK");
     }
     // Check if I am the oldest backup DS (I will no longer be part of the DS committee)
-    else if ((uint32_t)(m_consensusMyID + 1)
-             == m_mediator.m_DSCommitteeNetworkInfo.size())
+    else if ((uint32_t)(m_consensusMyID + 1) == m_mediator.m_DSCommittee.size())
```

Comment:
Not sure why the casting to uint32 is required here.

Reply:
Without the cast, we get -Werror=sign-compare (I guess the arithmetic on LHS results in a signed value by default)